src/Http/Client.php

```diff
@@ -227,6 +227,30 @@ public function addCookie(CookieInterface $cookie)
         return $this;
     }
 
+    /**
+     * Pass a string url to set the base domain and scheme for the request.
+     * Url must parse into a scheme and host.
+     *
+     * @param  string  $url A string url e.g. https://example.com
+     * @return self
+     */
+    public static function createScopedClientFromUrl(string $url): self
```

Comment:
The documented type is `static`, and there should be no return typehint on these methods.